src/test/java/henry/ParserTest.java

```diff
@@ -0,0 +1,23 @@
+package henry;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+import command.Command;
+
+public class ParserTest {
+
+    @Test
+    public void parseTest() {
```

Comment:
Could have a clearer naming of test eg. successfulParseTest()

Reply:
Modified in latest.